camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeperPlanner.java

```diff
@@ -63,15 +69,21 @@ private boolean shouldReprocess(FileSystem fs, Path source, long destinationModT
     return false;
   }
 
-  private class HiddenFilter implements PathFilter {
-    @Override
-    public boolean accept(Path path) {
-      String name = path.getName();
-      if (name.startsWith("_") || name.startsWith(".")) {
-        return false;
-      }
+  protected String pathListToCommaSeperated(List<Path> list) {
+    ArrayList<Path> tmpList = new ArrayList<Path>();
+    tmpList.addAll(list);
 
-      return true;
+    StringBuilder sb = new StringBuilder(tmpList.get(0).toString());
```

Comment:
This method is moved here from subclass CamusSweeperDatePartitionPlanner, since I need to use it in another subclass CamusHourlySweeprPlanner. I agree this can be refactored, but so are lots of other places in the old Camus code, so I'm not planning to do it.